services/webapp/src/features/posts/containers/AuthorDetails.jsx

```diff
@@ -51,7 +51,7 @@ class AuthorDetails extends React.PureComponent {
 
     render () {
         return this.props.isReady
-            ? <Content {...this.props} />
+            ? <Link to={`/users/${this.props.authorId}/details`}><Content {...this.props} /></Link>
```

Comment:
I think you are 90% close to getting this right.

Take a look at the definition of the "Content" component. It's few lines above.
Inside there we already have a link for the email, so this implementation ends up in a nested link that is not good.

Can you move the Link just around the author's username?

Reply:
Thank for your feedback. I have update it